modules/consignment-api/variables.tf

```diff
@@ -37,4 +37,8 @@ variable "dns_zone_name_trimmed" {}
 variable "ip_whitelist" {
   description = "IP addresses allowed to access"
   default     = ["0.0.0.0/0"]
+}
+
+variable "create_users_security_group" {
```

Comment:
Maybe add `_id` onto the end of the variable name to make clear an id value

Reply:
Yep,  good point, I've changed it.